pkg/download/release_matcher_test.go

```diff
@@ -31,8 +31,6 @@ func TestOsArch(t *testing.T) {
 		{"soluble_v0.4.19_linux_386.tar.gz", "linux", "386", true},
 		{"trivy_0.12.0_Linux-64bit.tar.gz", "linux", "amd64", true},
 		{"trivy_0.12.0_macOS-64bit.tar.gz", "darwin", "amd64", true},
-		{"terrascan_1.1.0_Linux_i386.tar.gz", "linux", "386", true},
```

Comment:
You can leave these test cases here as it is testing something.

Reply:
Otherwise looks fine.

Reply:
I’ve popped those back in, my thinking was this case would be irrelevant	now, but I see its still good for testing all the “matching” functions